src/__internal__/popover/popover.component.tsx

```diff
@@ -64,6 +64,8 @@ const Popover = ({
 
   if (!elementDOM.current && !disablePortal) {
     elementDOM.current = document.createElement("div");
+    elementDOM.current.setAttribute("role", "region");
+    elementDOM.current.setAttribute("aria-label", "ariaLabel-example");
```

Comment:
Can you assign a unique aira-label or set aria-labelledby for it to follow a unique Id?